spec/features/parks/01_index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+#
+#
+# I want to view a list of items
+# So that I can pick items to review
+
+
+# TO DO : As an authenticated user
+
+feature "users can view parks" do
+  before do
+    # user = FactoryGirl.create(:user)
+
+  end
+
+  scenario "visitor signs in and views parks on index page" do
+
+    parks = FactoryGirl.create_list(:park, 2)
+    visit root_path
+
+    # sign_in_as(user)
```

Comment:
it's part of our todo, once we have users merged, other pair is working on it now.
then we'll uncomment it and user auth. will be added to the test